PP_APP/server/server/settings.py

```diff
@@ -35,18 +35,18 @@
 
 
 # Application definition
-
 INSTALLED_APPS = [
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'django_cleanup',
     'rest_framework',
     'rest_framework.authtoken',
+    'rest_framework_datatables',
```

Comment:
This is missing from _requirements.txt_ so manage.py crashes. I was able to fix it by manually installing it but please update _requirements.txt_ so it gets installed automatically.